src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -37,9 +38,11 @@ class SaveInProgressIntro extends React.Component {
     );
     if (login.currentlyLoggedIn) {
       if (savedForm) {
+        const lastUpdated =
+          savedForm.lastUpdated || _.get('metadata.lastUpdated', savedForm);
         const savedAt = this.props.lastSavedDate
           ? moment(this.props.lastSavedDate)
-          : moment.unix(savedForm.lastUpdated);
```

Comment:
the component is using `savedForm.lastUpdated` but the unit test looks for `savedForm.metadata.lastUpdated`. I don't know which one of these is correct so I added some logic to use the metadata if the `savedForm.lastUpdated` doesn't exist.

Reply:
I think that's ok. The different dates in here are confusing, I'm not sure why we have so many similar ones.